src/avatar/Avatar.tsx

```diff
@@ -0,0 +1,35 @@
+import "./_avatar.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+export interface AvatarProps {
+  alt: string;
+  size?: {
+    width: string;
+    height: string;
+  };
+  placeholderSrc: string;
+  src?: string | null;
+  customClassName?: string;
+}
+
+function Avatar({alt, size, src, customClassName, placeholderSrc}: AvatarProps) {
+  const [shouldDisplayPlaceholder, setPlaceholderVisibility] = useState(true);
+
+  return (
+    <img
+      src={shouldDisplayPlaceholder ? placeholderSrc : src || placeholderSrc}
+      alt={alt}
+      style={{...size}}
+      className={classNames("avatar", customClassName)}
+      onLoad={handleHidePlaceholder}
+    />
+  );
+
+  function handleHidePlaceholder() {
```

Comment:
I think `handleImageLoad()` may be a more descriptive name for this function, because here we are handling the load event of the image
```suggestion
  function handleImageLoad() {
```

Reply:
Thanks for your suggestions. I applied this changes [@this commit ](https://github.com/Hipo/react-ui-toolkit/pull/55/commits/d59c58b6d37a862c9972aa296516d09122df7e3f) :pray: